src/main/java/com/r307/arbitrader/service/TradingService.java

```diff
@@ -179,8 +180,10 @@ private void enterPosition(Spread spread) {
         LOGGER.debug("Short scale: {}", shortScale);
         LOGGER.debug("Long ticker ASK: {}", spread.getLongTicker().getAsk());
         LOGGER.debug("Short ticker BID: {}", spread.getShortTicker().getBid());
-        LOGGER.debug("Long fee percent: {}", longFeePercent);
-        LOGGER.debug("Short fee percent: {}", shortFeePercent);
+        LOGGER.debug("Long fee percent: {}", longExchangeFee.getTradeFee());
+        if (shortExchangeFee.getMarginFee().isPresent()) {
+            LOGGER.debug("Short fee percent: {}", shortExchangeFee.getMarginFee().get());
```

Comment:
This should be `shortExchangeFee.getTradeFee().add(shortExchangeFee.getMarginFee().get())`. The trade fee and margin fees need to get added together.